gensim/similarity_learning/models/drmm_tks.py

```diff
@@ -0,0 +1,108 @@
+import keras.backend as K
+from keras.models import Model
+from keras.layers import Input, Embedding, Dot, Dense, Lambda, Reshape, Dropout
+from keras.activations import softmax
+
+
+class DRMM_TKS:
```

Comment:
Looks like it's time to move it into our interfaces:
- Inherit `utils.SaveLoad` class for save/load
- Method for training (don't forget about streaming)
- Looks like `build` method can be called in `__init__` and should be hidden from user.